var/spack/repos/builtin/packages/rocm-smi/package.py

```diff
@@ -11,20 +11,25 @@
 
 class RocmSmi(MakefilePackage):
     """This tool exposes functionality for clock and temperature
-       management of your ROCm enabled system"""
+       management of your ROCm enabled system
+
+       Note: According to its README, after ROCm 3.9, this project
+             moved to
```

Comment:
Simplifying, because we have the authors here:
```suggestion
       Note: After ROCm 3.9, this project moved to
```